as/thrift.js

```diff
@@ -66,17 +66,13 @@ function TChannelAsThrift(opts) {
         'channel must be provided with isHealthy');
 
     if (self.isHealthy) {
-        fs.readFile(path.join(__dirname, 'meta.thrift'), 'utf8', registerHealthCheck);
+        var thriftSource = fs.readFileSync(
```

Comment:
sync io in constructor is sad, can we instead inject the content thru opts, and then provide a `TChannelThift.somethingSync` convenience?